packages/widget-chat/src/actions/flags.js

```diff
@@ -53,11 +86,14 @@ function actionUpdateFlag(flag) {
  */
 export function fetchFlags(spark) {
   return (dispatch) => {
-    dispatch(actionBeginReceiveFlags());
-    spark.flag.list()
-      .then((flags) => {
-        dispatch(actionReceiveFlags(flags));
-      });
+    dispatch(flagsRequestBegin());
```

Comment:
Same as above comment. To fit the rest of the pattern it may be good to change this to `requestFlagsBegin`